roles/rabbitmq/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# set to [] to disable.
+rmq_plugins:
```

Comment:
Please prefix the var by role name to follow the best practices, eg `rabbitmq_plugins`